### ISSUE TITLE
SQL error while installating Ubuntu environment

### ISSUE THREAD

**Issue by pchaigno** [markdenihan/owaspSecurityShepherd#131]
I'm following the guidelines to setup the Ubuntu environment.
I get the following SQL error when doing `mysql -u root -e "source coreSchema.sql" --force -p`.

```
ERROR 1452 (23000) at line 1474 in file: 'coreSchema.sql':
Cannot add or update a child row:
a foreign key constraint fails (`core`.`cheatsheet`, CONSTRAINT `fk_CheatSheet_Modules1` FOREIGN KEY (`moduleId`) REFERENCES `modules` (`moduleId`) ON DELETE CASCADE ON UPDATE CASCADE)
```

**Comment by markdenihan** [comment]
We'll need to look into this. Hopefully it's a cheat sheet from a removed
level rather than a level with no cheat sheet

**Comment by jasoneflood** [comment]
SET FOREIGN_KEY_CHECKS = 0;

You might try adding this to the start of your install script.

The last line should set it back to 1.

It will mean it will still be broken, but at least it won't road block
anyone...

Just a thought.

**Comment by pchaigno** [comment]
@jasoneflood It does fix the issue with `coreSchema.sql` but then I've a new one with `moduleSchemas.sql` (it might be related):
```
ERROR 1133 (42000) at line 1154 in file: 'moduleSchemas.sql': Can't find any matching row in the user table
```

**Comment by markdenihan** [comment]
Ok so the error in comment one (Foreign Key) was actually resolved in commit 86ff6afa742dc6a8502e6e97590b8541c48dcefb last week. The script isn't broken by the error, the application isn't missing any data. It will be in the next patch/update

The Second error is related to unused historic users that had not been removed from the script. The Errors won't impact the install / application experience. Removed the unused users and their schemas to the latest commit c5895be01be571276614cfeae4309e6969e4a018. Wasn't able to reproduce the error afterwards

Thanks for calling those out @pchaigno !

**Comment by pchaigno** [comment]
Awesome!
